src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ExpressRouteCircuitTests.ps1

```diff
@@ -546,53 +546,111 @@ Tests ExpressRouteCircuitConnectionCRUD.
 #>
 function Test-ExpressRouteCircuitConnectionCRUD
 {
-	$circuitName = "dedharcktpeer"
-	$groupName = "dedharpspeer"
-	$peerCircuitId = "/subscriptions/99c33776-9f4e-4e58-abe8-9263db1b9c6e/resourceGroups/dedharpsinit/providers/Microsoft.Network/expressRouteCircuits/dedhar-cktinit/peerings/AzurePrivatePeering"
-	$addressPrefix = "60.0.0.0/29"
-	$authorizationKey = "aaf441cf-4409-48ee-8e2d-a39cc7e428a8"
-	$connectionName = "transit"
+	$initCircuitName = "init_ckt"
```

Comment:
Suppose hardcoded values are used as there is provisioned set of resources on the subscription? (usual way to get random test resource names is [like this](https://github.com/Azure/azure-powershell/blob/3d2ea32d14189160434d2893661e498e0b615d3f/src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ExpressRouteCircuitTests.ps1#L95)).
So once we enable BrkTest flag it could be updated to common approach. @dedhar, please confirm.

Reply:
In this test we need to create 2 sets of circuits and private peerings. Is there a way to accept multiple circuit names as input? Who specifies command line parameters ? All the other Express Route test cases require only one circuit.

Reply:
Found a way to do this inn wan tests. Updating PR now